packages/esm-patient-forms-app/src/hooks/use-forms-context.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { createContext, useContext, useState, useMemo } from 'react';
+
+interface FormsContextState {
+  pageSize: number;
+  currentPage: number;
+  searchTerm: string;
+  setPageSize: (size: number) => void;
+  setCurrentPage: (page: number) => void;
+  setSearchTerm: (term: string) => void;
+}
+
+const FormsContext = createContext<FormsContextState | undefined>(undefined);
+
+export const FormsProvider: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  const [pageSize, setPageSize] = useState(50);
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
These feel like they should be props passed into the provider rather than hard-coding defaults as state.